addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -191,7 +179,66 @@ const FontFamilyPickerUserValueWidget = SelectUserValueWidget.extend({
         });
         this.allFonts = [];
 
-        await this._super(...arguments);
+        const fontsToLoad = [];
+        for (const font of this.googleFonts) {
+            const fontURL = `https://fonts.googleapis.com/css?family=${encodeURIComponent(font).replace(/%20/g, '+')}`;
+            fontsToLoad.push(fontURL);
+        }
+        for (const font of this.googleLocalFonts) {
+            const attachmentId = font.split(/\s*:\s*/)[1];
+            const fontURL = `/web/content/${encodeURIComponent(attachmentId)}`;
+            fontsToLoad.push(fontURL);
+        }
+        const proms = fontsToLoad.map(async fontURL => loadCSS(fontURL));
+        const fontsLoadingProm = Promise.all(proms);
+
+        this._fonts = [];
+        const themeFontsNb = nbFonts - (this.googleLocalFonts.length + this.googleFonts.length);
+        const googleLocalFontsOffset = nbFonts - this.googleLocalFonts.length;
+        for (let fontNb = 0; fontNb < nbFonts; fontNb++) {
+            const realFontNb = fontNb + 1;
+            const fontKey = weUtils.getCSSVariableValue(`font-number-${realFontNb}`, style);
+            this.allFonts.push(fontKey);
+            let fontName = fontKey.slice(1, -1); // Unquote
+            let fontFamily = fontName;
+            const isSystemFonts = fontName === "SYSTEM_FONTS";
+            if (isSystemFonts) {
+                fontName = _t("System Fonts");
```

Comment:
This was copied from WeFontFamilyPicker widget, but the code was not removed, shouldn't it be?

Reply:
I actually forgot to get rid of the component's `start` method which did build the buttons "manually" - because this is now done in the template using the data structure built in the `FontFamilyUserValue`.